application/controllers/collection_instrument.py

```diff
@@ -354,24 +365,6 @@ def _find_or_create_business(ru_ref, session):
             business = BusinessModel(ru_ref=ru_ref)
         return business
 
-    @staticmethod
-    def _create_seft_file(instrument_id, file):
-        """
```

Comment:
this needs to stay temporarily otherwise the old database mechanism will no longer work